myuw/templates/handlebars/card/schedule/visual.html

```diff
@@ -9,27 +9,28 @@ <h3 id="quarter-info">{{ titleFormatTerm term_string }} Schedule</h3>
     </h3>
 
     {{#if has_eos_dates}}
-        <div style="margin-top: 30px; margin-bottom: 25px">
+        <div class="meeting-eos-sessions">
         {{#each eos_sections}}
-            <p><i>{{curriculum_abbr}} {{course_number}} {{ section_id }} meeting times:&nbsp;
-            {{#each meetings}}
-                <span>{{#not_first @index}},&nbsp;{{/not_first}}
-                    {{#if eos_start_date}}
-                        {{toMonthDay eos_start_date}}{{#unless start_end_same}} &ndash; {{toMonthDay eos_end_date}}{{/unless}}
-                        {{#if wont_meet}}
-                            (No meeting)
-                        {{else}}
-                            {{#if start_time}}({{formatTime start_time}} &ndash; {{formatTimeAMPM end_time}}){{/if}}
-                        {{/if}} 
-                    {{/if}}
-                </span>
-            {{/each}}
-            </i></p>    
+            <h4>{{curriculum_abbr}} {{course_number}} {{ section_id }} meeting times:&nbsp;
```

Comment:
This <h4> missing a ending tag.
Also you may want to change it to 
<h4 class="sr-only">Section Meeting with Dates</h4>